CameraOverlay/data.py

```diff
@@ -0,0 +1,89 @@
+import time
+from typing import Any, Optional
+import topics as topics
+
+V2_DATA_TOPICS = [
+    str(topics.DAS.data),
+    str(topics.PowerModel.recommended_sp),
+    str(topics.PowerModel.predicted_max_speed),
+    str(topics.PowerModel.plan_name),
+]
+
+V3_MESSAGE = [
+    str(topics.DAShboard.receive_message)
+]
+
+class Data:
+
```

Comment:
You're gonna hate me. But can you also include docstrings? Up to you with what standard you use! I wouldn't suggest numpy docstrings. Anything else would be good though!

The return types in each method is pretty handy though.